lib/view_resource/filter.dart

```diff
@@ -5,6 +5,9 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
 import 'package:web_app/events/schedule.dart';
 
+// Time
+import '../common.dart';
```

Comment:
I was thinking about standardizing towards absolute imports and away from relative imports -- i.e., `import 'package:web_app/common.dart';` One advantage being you don't have to change a module's imports if you ever move a module into or out of a folder. What do you think?